README.md

```diff
@@ -136,12 +136,11 @@ See [make-defaults.mk](make-defaults.mk) for a full list of manually configurabl
 
 Official releases of DC/OS can be found at <http://dcos.io/releases/>.
 
-By default, `./configure --auto` downloads the latest **stable** version of DC/OS.
+By default, `./configure` downloads the latest **stable** version of DC/OS.
 
-To use a different version, run `./configure` without the `--auto` option and specify the path to the installer.
-When running `configure`, the installer file must be located in (or under) the `dcos-docker` directory.
+To use a different version, run `./configure` (without `--auto`) and specify the local path to the installer when prompted.
```

Comment:
Do we still need "(without `--auto`)"?

Reply:
debatable... if you use auto it doesn't allow you to respond to the prompt...

Reply:
k, i removed it as unnecessary